practicals/large_language_models.ipynb

```diff
@@ -16,7 +16,7 @@
         "\n",
```

Comment:
<p>nit: Can we keep the prompt I had here?  And add a note / task asking people to modify the prompt? Fine by me to keep this if you prefer ;)</p><p> </p><pre class="ql-syntax" spellcheck="false">formatted_artist_name = artist_name.replace('-', ' ').title()
prompt = f'This is a song by {formatted_artist_name}. It goes like this:\n\n' # @param
print(prompt)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2023/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='144'/>

Reply:
<p>Whoop forgot to change it back! Changing again</p><p> </p>